pkg/cache/v3/snapshot.go

```diff
@@ -266,3 +269,30 @@ func (s *Snapshot) ConstructVersionMap() error {
 
 	return nil
 }
+
+// Respond to a delta watch with the provided snapshot value. If the response is nil, there has been no state change.
+func (s *Snapshot) respondDelta(ctx context.Context, request *DeltaRequest, value chan DeltaResponse, state stream.StreamState, log log.Logger) (*RawDeltaResponse, error) {
+	resources := &resourceContainer{
```

Comment:
Does this need to be a pointer? Maps and slices are pointers under the hood anyways so this might be redundant

Reply:
yep, removed pointer